controlpanel/api/fixtures/status_page_events.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Fixture file so that I can test how the events are displayed locally, without having to create test events in PagerDuty. Here is how it looks on the frontend after loading the fixtures to the DB:
<img width="4064" height="2334" alt="image" src="https://github.com/user-attachments/assets/3bb85bf7-27b9-448e-94f7-a5d75146c895" />